### PR TITLE
Add project report page

### PR DESCRIPTION
**Pull request by McSinyx**
Still need documentation and tests, though early reviews are appreciated.

### CODE REVIEW

File: tests/assets/rethinkdump/test/projects.json [hunk 1]
```diff
@@ -13,6 +13,11 @@
     "description": "Consectetur neque ut adipisci labore. Dolor velit porro adipisci. Consectetur sed ipsum neque porro etincidunt dolor ut. Ipsum voluptatem dolorem dolorem etincidunt. Etincidunt ut dolore velit numquam quisquam. Consectetur non non non quiquia consectetur velit velit. Velit neque ipsum aliquam ut adipisci sit. Tempora voluptatem aliquam non dolore eius quaerat.",
     "id": "7fee498b-98ac-4173-b600-e94618f9ea1f",
     "name": "Amet sed porro ut quisquam velit quisquam modi",
+    "report": {
+      "comment": "",
+      "grade": -1,
```
Comment (Huy-Ngo) [inline tests/assets/rethinkdump/test/projects.json:18]: This means the report is not graded yet?

File: tests/assets/rethinkdump/test/projects.json [hunk 2]
```diff
@@ -125,6 +130,13 @@
     "description": "Quaerat quaerat dolor labore. Quisquam voluptatem tempora quiquia ut. Dolore quisquam tempora adipisci. Magnam eius sit neque quiquia est ipsum. Consectetur quiquia sit eius adipisci est. Adipisci tempora aliquam voluptatem etincidunt dolorem dolor. Ut ipsum quiquia etincidunt neque adipisci porro consectetur.",
     "id": "be7a04b8-650f-41dc-912b-10d225baff29",
     "name": "Eius aliquam modi modi sit adipisci dolor",
+    "report": {
+      "comment": "Très bien!",
+      "grade": 22,
```
Comment (Huy-Ngo) [inline tests/assets/rethinkdump/test/projects.json:135]: Why is this out of range.

File: tests/assets/rethinkdump/test/projects.json (hunk 1 above)
Comment (McSinyx) [reply tests/assets/rethinkdump/test/projects.json:18]: I guess LOL, it's my default placeholder, have not thought much bout it.

File: tests/assets/rethinkdump/test/projects.json (hunk 2 above)
Comment (McSinyx) [reply tests/assets/rethinkdump/test/projects.json:135]: [Which range](https://en.wikipedia.org/wiki/Ahuy) :upside_down_face:

Comment (Huy-Ngo) [reply tests/assets/rethinkdump/test/projects.json:135]: USTH uses grade on scale of 20 (like in France). /shrug

Comment (McSinyx) [reply tests/assets/rethinkdump/test/projects.json]: Addressed.

**Review by Huy-Ngo — APPROVED** [review]

**Comment by McSinyx** [comment]
Thanks!

**Pull request merged**